sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -59,6 +59,12 @@ def _get_dummy_metric_params_list(metric: str, n_features: int):
     if metric == "seuclidean":
         return [dict(V=rng.rand(n_features))]
 
+    if metric == "mahalanobis":
```

Comment:
Seems like this is unreachable code

Reply:
I can't remember why this was introduced.

Removing it from now via 16d777f0c8885c5e2c2799b602d524446fa705ad.